lib/framework/view/page.dart

```diff
@@ -554,18 +556,30 @@ class PageState extends State<Page> {
   Widget? _buildFooter(ScopeManager scopeManager, SinglePageModel pageModel) {
     // Footer can only take 1 child by our design. Ignore the rest
     if (pageModel.footer != null && pageModel.footer!.children.isNotEmpty) {
+      final footerStyles = pageModel.footer?.styles;
+      final boxController = BoxController()
+        ..padding = Utils.getInsets(footerStyles?['padding'],
+            fallback:
+                const EdgeInsets.only(left: 16, right: 16, top: 16, bottom: 32))
+        ..margin = Utils.optionalInsets(footerStyles?['margin'])
+        ..width = Utils.optionalInt(footerStyles?['width'])
+        ..height = Utils.getInt(footerStyles?['height'], fallback: 110)
```

Comment:
remove all the fallbacks. It'll work still right?

Reply:
Yes, It'll work. I removed it